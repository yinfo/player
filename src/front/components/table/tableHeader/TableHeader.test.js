// @flow

import React from 'react';
import { shallow } from 'enzyme';
import TableHeader from './TableHeader';

describe('TableHeader component', () => {
  it('renders as expected', () => {
    const component = shallow(<TableHeader>{<p>a child</p>}</TableHeader>);
    expect(component).toMatchSnapshot();
  });
});
