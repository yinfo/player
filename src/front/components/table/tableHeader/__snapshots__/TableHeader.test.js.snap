// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeader component renders as expected 1`] = `ShallowWrapper {}`;
